### ISSUE TITLE
[Bug]: Details components - describe component (Level AAA - Accessibility)

### ISSUE THREAD

**Issue by joepercival** [ONSdigital/design-system#2844]
### What happened?

This issue was raised on the Statistical Methods Library static website: [Link to details components - describe component ticket](https://jira.ons.gov.uk/browse/SPP-9555)

### What was the expected outcome?

See ticket

### How is this problem impacting your service?

See ticket

### What list of steps can be followed to reproduce the bug or issue?

See ticket

### What version of the ONS Design System are you using?

62.0.0

### What devices are you seeing the problem on?

- [X] Desktop
- [ ] Mobile
- [ ] Tablet

### What operating systems are you seeing the problem on?

- [ ] Windows
- [ ] macOS
- [ ] Android
- [ ] iOS
- [ ] Other

### What browsers are you seeing the problem on?

- [ ] Firefox
- [ ] Chrome
- [ ] Safari
- [ ] Microsoft Edge
- [ ] Microsoft Internet Explorer
- [ ] Other

**Comment by yatesn** [comment]
Investigate whether change should be made

**Comment by adi-unni** [comment]
@adi-unni
